### DISCUSSION TITLE
HA integration reports timeout

### DISCUSSION THREAD

**Discussion by aircooledbeemer** [timmo001/system-bridge#2185] [Help]
I have systembridge running on Debian 11 and can see the system tray and data under it.
Home assistant supervised is running on the same box, setting up the integration reports success, however then it stays trying to initialize with the following message:
Retrying setup: Timed out waiting for XXX  (address).

Log says
Logger: homeassistant
Source: components/system_bridge/coordinator.py:150
First occurred: 10:27:58 (2 occurrences)
Last logged: 10:28:33
Error doing job: Task exception was never retrieved

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/system_bridge/coordinator.py", line 150, in _listen_for_data
    await self.websocket_client.listen(callback=self.async_handle_module)
  File "/usr/local/lib/python3.10/site-packages/systembridgeconnector/websocket_client.py", line 492, in listen
    await self.listen_for_messages(callback=_callback_message)
  File "/usr/local/lib/python3.10/site-packages/systembridgeconnector/websocket_client.py", line 507, in listen_for_messages
    await callback(message)
  File "/usr/local/lib/python3.10/site-packages/systembridgeconnector/websocket_client.py", line 480, in _callback_message
    model(**message[EVENT_DATA]),
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 2 validation errors for Display
displays
  field required (type=value_error.missing)
last_updated -> displays
  field required (type=value_error.missing)

 Logger: homeassistant.config_entries
Source: config_entries.py:1113
First occurred: 10:28:28 (1 occurrences)
Last logged: 10:28:28
Config entry 'name' for system_bridge integration not ready yet: Timed out waiting for XXX (ip).; Retrying in background

Any idea what could go wrong here?

Many thanks!

**Comment by lexiismadd** [comment]
I'm also having the same issue

**Comment by jd1** [comment]
Same problem here. There is already a issue for the home assistant integration: https://github.com/home-assistant/core/issues/77731

**Reply by jd1** [reply]
With the new HA 2023.4 release the timeout issue is gone, but it looks like there are still some issues that entities are not updating.
